doc/source/dev/contributor/pep8.rst

```diff
@@ -39,6 +39,22 @@ compliance before pushing your code:
    right, consider introducing (and then removing) a small PEP8 issue in
    the target file, such as a line that is over 79 characters long.
 
+-  Before sending a Pull Request, we recommend running the lint tests only
+   for the changes you've made in your feature branch. This will mimic
+   the continuous integration linting checks setup on GitHub.
```

Comment:
If you agree that it belongs here, I might add something like:

> Generally, you are not responsible for fixing pre-existing PEP8 issues in 
   the codebase. In fact, it is typically suggested that you leave existing
   issues alone unless they are parts of the code you're already modifying.
   This practice ensures that the code base is gradually cleaned up over 
   time without spending precious review time on style-only cleanups.